helpers/index.ts

```diff
@@ -0,0 +1 @@
+export { default as isWebThreeInstalled } from "./isWebThreeInstalled";
```

Comment:
Don't make this export barrel or import lines will become pretty large when we more than a few helpers

Reply:
I do. That's exactly the point when dealing with more than one helper module. This solves the problem on importing default named modules at components. Instead of this:
```jsx
import module1 from "helpers/module1";
import module2 from "helpers/module2";
import module3 from "helpers/module3"
```

the current structure enhances import calls, like:
```jsx
import { module1, module2, module3 } from "helpers";
```

Reply:
@soujvnunes that import helpers line will get big in a matter of seconds, besides, the only function that exists is `isWebThreeInstalled`, no point making an export barrel for one function. But even so, you exported that function as a `default` meaning _you want_ people to consume it as 

```
import function from `file.ts`
```

But then you went and _import the deconstruction of the module_ only to export that as isWebThreeInstalled again, when all you had to do was:

```
import isWebThreeInstalled from `@helpers/is-web-3-installed.ts
```

And all would be cool with everything because the person reading it didn't have to jump through two files to get to the source of the file.

Reply:
Yup! And I think we will not do this in most components besides where currently it is. I'm gonna remove the indexation